src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/repositories/DraftAdjudicationRepository.kt

```diff
@@ -1,12 +1,12 @@
 package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.repositories
-
-import org.springframework.data.jpa.repository.Query
 import org.springframework.data.repository.CrudRepository
 import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.entities.DraftAdjudication
+import java.time.LocalDateTime
 
 interface DraftAdjudicationRepository : CrudRepository<DraftAdjudication, Long> {
   fun save(draftAdjudication: DraftAdjudication?): DraftAdjudication
 
-  @Query("SELECT da from DraftAdjudication da where da.agencyId = :agencyId and da.createdByUserId = :username and da.reportNumber IS NULL")
-  fun findUnsubmittedByAgencyIdAndCreatedByUserId(agencyId: String, username: String): List<DraftAdjudication>
+  fun findDraftAdjudicationByAgencyIdAndCreatedByUserIdAndReportNumberIsNull(agencyId: String, username: String): List<DraftAdjudication>
```

Comment:
Probably should have done this in a different pull request - apologies. (We prefer the spring naming conventions over annotations?)